webapp-django/crashstats/crashstats/migrations/0010_1506907_product_migration.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-11-14 23:26
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.db import connection, migrations
+
+
+def copy_products(apps, schema_editor):
+    Product = apps.get_model('crashstats', 'Product')
+    cursor = connection.cursor()
+
+    # Verify the table is there. If not, we don't need to do anything.
+    cursor.execute("""
+    SELECT relname FROM pg_catalog.pg_class
+    WHERE relname = 'products'
+    """)
+    row = cursor.fetchone()
+    if row is None:
+        print('no products table--nothing to do', end='')
+        return
+
+    # Second, pull all the data from it
+    columns = ['product_name', 'sort']
+    cursor.execute(
+        """
+        SELECT %(columns)s
+        FROM products
+        """ % {'columns': ', '.join(columns)}
+    )
+
+    # Third, insert it into the new table
+    insert_count = 0
+    for row in cursor.fetchall():
+        product = dict(zip(columns, row))
+        # NOTE(willkg): In the old table, we used sort=-1 to denote inactive
+        # products. In the new table, we have a dedicated field.
+        Product.objects.create(
+            product_name=product['product_name'],
+            sort=product['sort'] if product['sort'] != -1 else 100,
+            is_active=product['sort'] != -1
+        )
+        insert_count += 1
+    print('(inserted: %s)' % insert_count, end='')
```

Comment:
With other tables, we do a sql script and run it manually after a deploy. That means we have to do these switch-overs in multiple steps and multiple deploys. That's fine, but this table has like 10 things in it and it's easier to do it all in one fell swoop if we can and here, we can.